wasm2c/wasm-rt-impl.c

```diff
@@ -414,7 +414,7 @@ const char* wasm_rt_strerror(wasm_rt_trap_t trap) {
     case WASM_RT_TRAP_UNREACHABLE:
       return "Unreachable instruction executed";
     case WASM_RT_TRAP_CALL_INDIRECT:
-      return "Invalid call_indirect";
+      return "Invalid call_indirect or return_call_indirect";
```

Comment:
Is there a reason this error code has to be overloaded?

Reply:
I guess not, but it helps share a lot of code between `call_indirect` and `return_call_indirect`.

Reply:
ah, gotcha. Tbh, not a big deal. We can fix it if easy, or leave as is.